README.md

```diff
@@ -1 +1,2 @@
-# xblock-review
\ No newline at end of file
+# xblock-review
+This repository contains all of the code for the Review XBlock. The Review xBlock is designed to act as a review tool for learners in their edX course. The way it is implemented, the xBlock pulls from all problems a learner has previously loaded in the course and randomly selects 5 of them to reshow the learner. The idea behind reshowing problems the learner has seen is to simulate a review of the concepts the problems should be testing so in retrying the problem, they are reviewing their knowledge of the concept. These review problems contain a fresh state of the problem and are ungraded so learners are able to try them without fear of it harming their grade.
```

Comment:
This text could be improved. Maybe a tl;dr one-sentence summary? Screenshots? Multiple sections, like "License", "Testing", "How To Develop"?